.gitlab/includes/gcc9_cuda11_pipeline.yml

```
@@ -0,0 +1,73 @@
+# Copyright (c) 2023 ETH Zurich
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file BOOST_LICENSE_1_0.rst or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+include:
+  - local: '.gitlab/includes/common_pipeline.yml'
+  - local: '.gitlab/includes/common_spack_pipeline.yml'
+
+.variables_gcc9_cuda11_config:
+  variables:
+    ARCH: linux-ubuntu22.04-broadwell
+    BUILD_TYPE: Debug
+    COMPILER: gcc@9.3.0
+    CXXSTD: 17
+    SPACK_SPEC: "pika@main arch=$ARCH %${COMPILER} +cuda malloc=system cxxstd=$CXXSTD \
+                 ^boost@1.75.0 ^hwloc@2.0.3 ^cuda@11.2.0 ^fmt@9"
```

This should not be necessary now that you've updated the spack commit:
```suggestion
                 ^boost@1.75.0 ^hwloc@2.0.3 ^cuda@11.2.0"
```

---

All right sounds good. In that case there may be new/other problems with fmt 10 and CUDA, and we may have to extend the conflicts in the spack package.